### PR TITLE
[shopsys] add brands to frontend API

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We want to add everything to frontend API that could be displayed on frontend. This PR adds brands.
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Review by grossmannmartin — CHANGES_REQUESTED** [review]
nice work!

can you chek please my comments?

File: packages/frontend-api/apiary.apib [hunk 1]
```diff
@@ -2344,3 +2357,219 @@ Returns article detail
...
+
+### POST [POST]
+
+Returns complete list of brands
+
+- Request (application/json)
+
+    {
+        brands {
+            uuid
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply packages/frontend-api/apiary.apib:2371]: Well US does not say anything about that. Also when some eshop have side with brands list it mostly is not paginated.

File: packages/frontend-api/src/Model/Resolver/Products/ProductResolverMap.php [hunk 2]
```diff
@@ -113,6 +149,17 @@ protected function mapProduct(): array
             'isUsingStock' => function ($data) {
                 return $data instanceof Product ? $data->isUsingStock() : $data['is_using_stock'];
             },
+            'brand' => function ($data) {
+                if ($data instanceof Product) {
+                    return $data->getBrand();
+                }
+
+                if (is_numeric($data['brand'])) {
+                    return $this->brandFacade->getById($data['brand']);
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply packages/frontend-api/src/Model/Resolver/Products/ProductResolverMap.php]: fixed in 691724a42415505235252f2682caf7d027f3c89d

File: packages/frontend-api/apiary.apib (hunk 1 above)
[the remark this answers is not in the document]
Comment (grossmannmartin) [reply packages/frontend-api/apiary.apib:2371]: Okay 👍

**Review by grossmannmartin — APPROVED** [review]

**Review by sslt — APPROVED** [review]

**Pull request merged**